src/ResourceManagement/NotificationHubs/NotificationHubs.Tests/Tests/ScenarioTests.NamespaceTests.CRUDAuthorizationRules.cs

```diff
@@ -27,6 +27,7 @@ namespace NotificationHubs.Tests.ScenarioTests
     using System.Net;
     using System.Collections.Generic;
     using System.Linq;
+
     public partial class ScenarioTests 
     {
         [Fact]
```

Comment:
Can we split this test into multiple smaller tests. Rather have individual tests for Create/Delete/Update, because when this test fails, you will not know right away which operation failed.